### DISCUSSION TITLE
Experimental Rendering and Shader

### DISCUSSION THREAD

**Discussion by night2876** [xoreos/xoreos#591] [General]
Hi,

i've continued to work on the project and i'm at a good point. Now to update the graphics i would like to use vertex shader and fragment shader and use light and so on ... I read in the code that there is an experimental rendering system that work only with shader but when i switch to that don't work at all the screen is always black ... any idea?

Using the "standard" engine a lot of rendering don't use shader but glBegin(GL_QUANDS) .... glEnd() ...

This rendering system was finished or was only started? Could someone be intrested in give me same help to continue the work?

Best Regards.

Could someone give me a discord contatc to be more efficient in comunication ?

**Comment by DrMcCoy** [comment]
It was started and @mirv-sillyfish is working on it. You two might want to coordinate and work together on that if you'd like.

We're also in #xoreos on [Libera IRC](https://libera.chat/), @mirv-sillyfish is mirv_ there. Might be best for real-time communication, maybe.

**Reply by night2876** [reply]
Oh thanks i join gladly.
